packages/core/accounts/utils/toKernelPluginManager.ts

```diff
@@ -343,6 +375,9 @@ export async function toKernelPluginManager<
 
                 identifier: pad(getIdentifier(true), { size: 21, dir: "right" })
             }
+        },
+        signUserOperationWithActiveValidator: async (userOperation) => {
```

Comment:
Not sure about this function yet. We can have it like this for now. Might require another iteration of multi chain flow. But should be fine for now.